nls/platform.properties

```diff
@@ -53,6 +53,52 @@ application.argo.applicationset = This is a group of Argo applications generated
 application.type.acm = ACM application
 application.type.argo = Argo app set
 
+argo.create.title = Create Argo App set
```

Comment:
Capitalization of "app" does not match in these two cases. I am not sure which is correct - can we get ID and design to review all new strings?

Reply:
@KevinFCormier Looks like ID is fine with either app set or App set(https://coreos.slack.com/archives/CUHCCV19P/p1628523904005200), but yeah we probably need to align those two.

Reply:
I think lowercase `a` then in `app`. Generally in the UI I think we only capitalize the first word or proper nouns like Argo. So I think also `kubectl` rather than `Kubectl`. And `Revision type` rather than `Revision Type`.

Reply:
KK thanks for catching that!